et --hard HEAD

```diff
@@ -0,0 +1,255 @@
+
+    Update version in build guide
+
+[33mcommit d496080026f810d842257add14bcc824c7f5643c[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Wed Jul 21 12:42:43 2021 +0100
+
+    Upgrade to 2021.1
+
+[33mcommit 421953bd7d80d5396129ddd301137e1a5a53cf7f[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Sun Jul 18 18:36:26 2021 +0100
+
+    More raycast refactoring
+
+[33mcommit 75cadf0b91df410ad7d82bff6fb76a530e8e4e83[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Sun Jul 18 17:14:49 2021 +0100
+
+    Null check TOCTOU
+
+[33mcommit aa31522a480502a9d1e7b598cdc15b490bee7084[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Sun Jul 18 17:04:01 2021 +0100
+
+    Bop unity analytics
+
+[33mcommit 0b8014b3b673754cb2d2645189d291cc51041493[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Wed Jul 14 14:38:42 2021 +0100
+
+    Raycasting fixes
+
+[33mcommit b0dca1b6a18b39c0767718051cd178fee72f0044[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Wed Jul 14 13:12:30 2021 +0100
+
+    Add option to outline shader to offset mesh scaling
+
+[33mcommit 0712ab359bdca7d9629019a1b924eba244405569[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 16:45:45 2021 +0100
+
+    REEEEEEEEEEEEE brackets
+
+[33mcommit c8382edd8e4f76e26a0a8d7b0fe65d42a0c5a74d[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 15:42:01 2021 +0100
+
+    Fix swedish too
+
+[33mcommit ff4e5ae03f480f6ea0847e6ae3c80d841c54bda3[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 15:30:10 2021 +0100
+
+    Fix spanish locale code
+
+[33mcommit bacbdbfec3ed4df64ae379540203e48c5849bb66[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 14:21:38 2021 +0100
+
+    Update translations
+
+[33mcommit 3862d39050ebfaa57d8a9ab7c33b13183a8df02d[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 14:19:24 2021 +0100
+
+    Minor changes to strings
+
+[33mcommit efefca49a807a44e217a046bbad92987ed8452cb[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 14:19:24 2021 +0100
+
+    Minor changes to strings
+
+[33mcommit a9f0851f2db63354744d652948afaa25fdbf008e[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 11:12:49 2021 +0100
+
+    Wait for previous spectro generation to finish before generating another in song edit ui
+
+[33mcommit 6bf6a47d5f05466003531b5a47d72a2e13d65083[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 10:58:40 2021 +0100
+
+    Force placement note to not show arc
+
+[33mcommit 1c701f82edc1eab55904e15be41637058d4392b7[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 10:48:39 2021 +0100
+
+    Improve options menu resilience
+
+[33mcommit d561e80ee0f72ef1a53139106d64826367060cff[m
+Author: Thomas Cheyney <top_cat@thomasc.co.uk>
+Date:   Mon Jul 12 10:42:23 2021 +0100
+
+    Fix selected tab when returning to song li
```

Comment:
what is this file? i dont think its necessary here.

Reply:
Someone accidentally `git log > et --hard HEAD`

Reply:
Whoops. Not sure how I managed to accidentally do that but its been deleted